src/Phinx/Db/Adapter/SqlServerAdapter.php

```diff
@@ -1282,12 +1282,15 @@ protected function getIndexSqlDefinition(Index $index, $tableName)
             return $ret;
         }, $columnNames);
 
+        $includedColumns = $index->getInclude() ? sprintf(' INCLUDE ([%s])', implode('],[', $index->getInclude())) : '';
```

Comment:
Given the space in the query below, probably don't need the space before `INCLUDE` here